testapps/php56_extensions/composer.json

```diff
@@ -1,6 +1,33 @@
 {
     "require": {
         "php": "^5.6",
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "4.8.*",
```

Comment:
What happens if you specify older version of an extension? My guess is that it will fail meeting dependencies when running composer. Does it make sense if we fail faster than that, ideally with an actionable error message?